### ISSUE TITLE
is it possible function of custom css for markdown?

### ISSUE THREAD

**Issue by lowpolyegg** [marktext/marktext#3564]
### Description
i want change header background and header size in source code mode.
i found description of that function  in this forum.
According to the writing, it should be implemented in marktext0.17.1.

is it possible in now version?

### Versions

- MarkText:0.17.1
- Operating system:windows 10

**Comment by Jocs** [comment]
Yes, custom CSS is supported via the **`customCss`** field in **Preferences > Theme**. Closing as answered.
